charts/sonarqube-dce/README.md

```diff
@@ -201,8 +201,8 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `ApplicationNodes.prometheusExporter.httpsProxy` | HTTPS proxy for downloading JMX agent | `""` |
 | `ApplicationNodes.prometheusExporter.noProxy` | No proxy for downloading JMX agent | `""` |
 | `ApplicationNodes.prometheusExporter.securityContext` | Security context for downloading the jmx agent | see `values.yaml`|
-| `ApplicationNodes.plugins.install` | List of plugins to install | `[]` |
-| `ApplicationNodes.plugins.lib` | Plugins libray | `[]` |
+| `ApplicationNodes.plugins.install` | Link(s) to the plugin JARs to download and install | `[]` |
+| `ApplicationNodes.plugins.lib` | **DEPRECATED** Plugin libraries to download and install | `[]` |
```

Comment:
I'm not sure about the terminology here, we are saying it is deprecated, but at the same time we remove the behavior associated to it
I would say it is either we say it is deprecated and keep the behavior, then remove in the future, or remove entirely the possibility to set this value

Reply:
At the moment, this parameter is useless. It does not really contribute to installing the plugin. You can just assign a name to the jar using the `plugins.lib` parameter, which is not that useful in my opinion (why would need to rename the name of the plugin? 😃). The description is also not really clear, I've tried by myself and renaming the library downloaded from the `plugins.install` parameter is the only use you can do of `plugins.lib`. I was thinking of deprecating because people might still use `plugins.lib` but its functionality does not make sense in my opinion (I can keep the renaming capability during the deprecation period though)

Reply:
I speak I: Even if it is not doing a lot and have a very low change of having a functional impact, i would prefer to write it deprecated, while keeping the feature for few release so that we are not removing something without notice.

Reply:
Generally, I agree with @jCOTINEAU. Even if the feature is not meaningful, it would be better to keep it entirely while being deprecated. However, we decided to remove the feature completely (and not deprecate it) as this is a misuse of the sonaqube-installation guidelines and it was never really supported by the product.